websites/P/Prambors Radio/presence.ts

```diff
@@ -2,75 +2,43 @@ const presence = new Presence({
     clientId: "630428033966276612"
   }),
   strings = presence.getStrings({
+    play: "presence.playback.playing",
     pause: "presence.playback.paused"
   }),
   presenceData: PresenceData = {
     largeImageKey: "logo"
   };
 
-let timestamp: number;
+let startTimestamp: number;
 
 presence.on("UpdateData", async () => {
-  const pause = (await strings).pause;
-  if (
-    ["streaming.pramborsfm.com", "live.pramborsfm.com"].includes(
-      document.location.hostname
-    )
-  ) {
+  const { play, pause } = await strings;
+  if (document.location.hostname === "live.pramborsfm.com") {
```

Comment:
Whats the point of this check when you do not support `pramborsfm.com`?

Reply:
yes it doesnt support pramborsfm.com, but it supports www.pramborsfm.com.
I didn't do anything to the main page so it's just like this
![image](https://user-images.githubusercontent.com/52526068/135780687-c70bd1c9-270d-47c0-9921-96a5e60de422.png)

Reply:
why bother add that domain then?

Reply:
to show that you open www.pramborsfm.com 🤔

Reply:
hi, so i need to remove www.pramborsfm.com?

Reply:
If you have no support to it, yes

Reply:
weird that it was merged on my previous pr